framework/generated/generated_vulkan_struct_to_json.cpp

```diff
@@ -305,9 +305,7 @@ void FieldToJson(nlohmann::ordered_json& jdata, const Decoded_StdVideoEncodeH264
         const Decoded_StdVideoEncodeH264SliceHeaderFlags& meta_struct = *data;
         FieldToJson(jdata["direct_spatial_mv_pred_flag"], decoded_value.direct_spatial_mv_pred_flag, options);
         FieldToJson(jdata["num_ref_idx_active_override_flag"], decoded_value.num_ref_idx_active_override_flag, options);
-        FieldToJson(jdata["no_output_of_prior_pics_flag"], decoded_value.no_output_of_prior_pics_flag, options);
-        FieldToJson(jdata["adaptive_ref_pic_marking_mode_flag"], decoded_value.adaptive_ref_pic_marking_mode_flag, options);
-        FieldToJson(jdata["no_prior_references_available_flag"], decoded_value.no_prior_references_available_flag, options);
+        FieldToJson(jdata["reserved"], decoded_value.reserved, options);
```

Comment:
The struct lost flags?

Reply:
That happens sometimes.  That would break existing programs as well as break existing traces that were already captured.  Perhaps in the future we could record the extension's version number with any structs related to the extension, and that would at least allow us to throw a warning.  This usually happens only very early in an extension's lifecycle, so I'm not inclined to worry about it much, but if it became a problem we could introduce versioned struct encoders and decoders.